tests/parsers.test.js

```diff
@@ -0,0 +1,225 @@
+const parsers = require("../parsers");
+
+describe("Object parser", () => {
+  const properObject = {
+    stringParam: "first",
+    numberParam: 123,
+    arrayParam: [1,2,3,4],
+    objectParam: {
+      test: "param"
+    }
+  };
+
+  test("returns an argument if proper object passed", () => {
+    expect(parsers.object(properObject)).toBe(properObject);
+  });
+
+  test("returns parsed object if proper JSON passed", () => {
+    expect(parsers.object(JSON.stringify(properObject))).toStrictEqual(properObject);
```

Comment:
That's functionality duplication. Use a hardcoded string to see what kind of value we use and then compare against  hardcoded object to see what results we expect - like you already do when parsing numbers. This applies to all the tests.